presto-expressions/src/main/java/com/facebook/presto/expressions/DynamicFilters.java

```diff
@@ -223,11 +253,18 @@ public DynamicFilterExtractResult(List<RowExpression> staticConjuncts, List<Dyna
     {
         private final String id;
         private final RowExpression input;
+        private final OperatorType operator;
```

Comment:
hi @rzeyde-varada am inclined to use `OperatorType`, so that we could extend support for `+`, `*`, and other operators. what do you think? just add `flip` and `negate` functionalities to `OperatorType`

Reply:
> just add flip and negate functionalities to OperatorType

Sounds good.